src/game.js

```diff
@@ -1,7 +1,27 @@
 class Game {
   drawGrid() {
-    // Iteration 1
-    // Draw the grid
-    // https://p5js.org/reference/#/p5/line
+    for (let x = 0; x <= WIDTH; x += WIDTH / 10) {
+      for (let y = 0; y <= HEIGHT; y += HEIGHT / 10) {
+        strokeWeight(1);
+        line(x, 0, x, height);
+        line(0, y, width, y);
+      }
+    }
+  }
+
+  preload() {
+    this.player.preload();
+  }
+
+  constructor() {
+    this.player = new Player();
```

Comment:
Remember to pass the necessary arguments to your `Player` constructor here!

`this.player = new Player(0,0)`